daemon/cmd/daemon_main.go

```diff
@@ -466,9 +466,12 @@ func initializeFlags() {
 		option.FixedIdentityMapping, "Key-value for the fixed identity mapping which allows to use reserved label for fixed identities, e.g. 128=kv-store,129=kube-dns")
 	option.BindEnv(option.FixedIdentityMapping)
 
-	flags.Duration(option.IdentityChangeGracePeriod, defaults.IdentityChangeGracePeriod, "Time to wait before using new identity on endpoint identity change")
+	flags.Duration(option.IdentityChangeGracePeriod, defaults.IdentityChangeGracePeriod, "Time to wait before using new identity on endpoint identity change when k8s is not enabled")
```

Comment:
Is this meant to be changed or is this modifying the wrong parameter? This patch is about the `IdentityRestoreGracePeriod`, not the `IdentityChangeGracePeriod`.

Reply:
changed the wrong parameter, will fix.